ios/FluentUI/Vnext/List/List.swift

```diff
@@ -10,6 +10,7 @@ import SwiftUI
 @objc public class MSFListSectionState: NSObject, ObservableObject, Identifiable {
     public var id = UUID()
     @objc @Published public var cells: [MSFListCellState] = []
+    @objc @Published public var style: MSFHeaderFooterStyle = .headerPrimary
     @objc @Published public var title: String?
     @objc @Published public var hasDividers: Bool = false
```

Comment:
The cell view [hasDividers](https://github.com/microsoft/fluentui-apple/blob/5609a48bee3a4e4cdb0741c980f140a31ecec21e/ios/FluentUI/Vnext/List/ListCell.swift#L52) should follow this pattern too.

Can you make these changes?
- name hasDivider (singular for the cell)
- should be a published property of the state object